Source/Core/Core/Boot/AncastTypes.h

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+// Magic number
+#define ANCAST_MAGIC 0xEFA282D9
```

Comment:
`constexpr u32 ANCAST_MAGIC = 0xEFA282D9;`
(we generally prefer real, typed constexpr variables over C-style defines)